bin/ansible-connection

```diff
@@ -244,7 +244,7 @@ def main():
 
     if rc == 0:
         ssh = connection_loader.get('ssh', class_only=True)
-        cp = ssh._create_control_path(play_context.remote_addr, play_context.port, play_context.remote_user, play_context.connection)
+        cp = ssh._create_control_path(play_context.remote_addr, play_context.port, play_context.remote_user, play_context.connection, os.getppid())
```

Comment:
do we want the pid of ansibe-connection or the one from ansible/ansible-playbook that starts ansbile-connection?